lib/logsene-api.js

```diff
@@ -27,14 +27,26 @@ var request         = require('request'),
     stringify       = require('../lib/util').safeJsonStringify;
 
 
-var logseneEsHost = process.env.LOGSENE_ES_HOST,
-    logseneUri = process.env.LOGSENE_URI,
-    es;
+function getEsHost(region) {
+  if (region == "EU") {
```

Comment:
Should be do a case insensitive check?

Reply:
I integrated case-insensitive checks now.